packages/react-native-bridge/index.js

```diff
@@ -515,6 +515,22 @@ export function sendEventToHost( eventName, properties ) {
 	);
 }
 
+/**
+ * Shows Android's soft keyboard if there's a TextInput focused and
+ * the keyboard is hidden.
+ *
+ * @param {Object} options
```

Comment:
Nitpick: We could define the properties of the object:
```suggestion
 * @param {Object} options
 * @param {number} [options.delay] Delay to show soft keyboard.
```